Sources/WebAuthenticationUI/Resources/en.lproj/Localizable.strings

```diff
@@ -1,8 +1,4 @@
 /* WebAuthenticationError */
-"default_property_list_not_found_description" = "The default Okta.plist configuration file was not found.";
-"invalid_property_list_description" = "The configuration file named \"%@\" was invalid.";
-"cannot_parse_property_list_description" = "Could not parse the configuration file named \"%@\".";
-"missing_configuration_values_description" = "Web authentication is not configured properly.";
```

Comment:
Not completely understand the idea of using localizable strings in SDK. Are you going to translate all these strings? Or you want to delegate this to app developer?
How you plan to deploy localization strings using Cocoapods? Pack into resource bundle?

Reply:
One of the issues we've faced with client SDKs in the past is developers blindly printing errors to the user, which results in very sub-optimal developer experiences.  Either that, or the error messages themselves are confusing to developers as they debug the application.

However, I'm not a fan of hard-coding strings when it's avoidable, so I'm starting with localization of these errors.  We may decide to roll this back in the future.

As far as Cocoapods support is concerned, yes my plan is to roll these into a resource bundle.